pkg/csi/types/envvars.go

```diff
@@ -31,4 +31,17 @@ const (
 
 	// EnvInClusterClientBurst is the Burst for all clients to the API server
 	EnvInClusterClientBurst = "INCLUSTER_CLIENT_BURST"
+
+	// EnvVarEndpoint specifies the CSI endpoint for CSI driver.
+	EnvVarEndpoint = "CSI_ENDPOINT"
+
+	// EnvVarMode is the name of the environment variable used to specify
+	// the service mode of the plugin. Valid values are:
+	// * <empty>
+	// * controller
+	// * node
+	//
+	// If unset or set to an empty value, both controller and node services
+	// are activated. The identity service is always activated.
+	EnvVarMode = "X_CSI_MODE"
```

Comment:
IIUC this change wont require any changes to the YAML files?

Reply:
Yes, since we're not changing the environment variable name, it won't require change in yamls.

Reply:
Can we remove all X_CSI references to avoid ambiguity with gocsi